README.md

```diff
@@ -118,12 +118,12 @@ incorporating the desired functionality from this repository.
 To use directly, replace the Nginx config directory with this repository. for example:
 
 ```shell
-nginx stop
+nginx -s stop
 cd /etc
 mv nginx nginx-previous
 git clone https://github.com/h5bp/server-configs-nginx.git nginx
 # install-specific edits
-nginx start
+nginx
```

Comment:
Does the command `nginx -s start` work? If so, I would prefer an explicit command.

```suggestion
nginx -s start
```

Reply:
Found my answer on the official documentation. Can't be more explicit.

Reply:
As you probably figured out, unfortunately it does not:
```bash
erik@neutron:~$ nginx -s start
nginx: invalid option: "-s start"
```
The only signals accepted by `-s` are `stop`, `quit`, `reload`, and `reopen`. To start nginx, you just use `nginx` without any arguments.